src/platforms/hosted/platform.c

```diff
@@ -398,8 +398,10 @@ const char *platform_target_voltage(void)
 		return jlink_target_voltage_string();
 #endif
 
+	case PROBE_TYPE_CMSIS_DAP:
+	case PROBE_TYPE_GPIOD:
```

Comment:
Not sure it's useful to list these out explicitly as that's what `default:` means here after all - noting that this is the handling for not-BMP when in a `HOSTED_BMP_ONLY=1` build.